### ISSUE TITLE
A regression in v3 crashes the process

### ISSUE THREAD

**Issue by fabiospampinato** [paulmillr/chokidar#838]
**Describe the bug**

Renaming a file to something involving the `∕` character (which isn't actually the forward slash character), makes the process crash.

**Versions (please complete the following information):**
 - Chokidar version: 3.0.0
 - Node version: 12.0.0 and 10.11.0
 - OS version: macOS 10.14

**To Reproduce**

```js
var chokidar = require('chokidar');
var fs = require('fs');
chokidar.watch('test*', {}).on('all', (event, path) => {
  console.log(event, path);
});
fs.writeFileSync('test.txt', 'testing 1');
fs.renameSync('test.txt', 'test∕asd.txt');
```

The thing crashes with a segmentation fault, it was working fine in v2.1.5.

**Comment by cekvenich** [comment]
Is fs. crashing or chokidar. What you wrote looks like fs and node issue.

**Comment by fabiospampinato** [comment]
If chokidar isn't listening it doesn't crash so I'd say it's an issue with chokidar (plus no crash in v2.1.5 also).

**Comment by paulmillr** [comment]
We can't reproduce this.

**Comment by paulmillr** [comment]
actually, I just did.

**Comment by pipobscure** [comment]
Thanks! Can you share the reproduction steps/scripts

**Comment by paulmillr** [comment]
@pipobscure just use the code in the OP post

**Comment by pipobscure** [comment]
Sorry, didn’t see (reading on phone). Will look at it tomorrow morning.

**Comment by astoilkov** [comment]
We stumbled upon the bug as well. I can also add that it happens for every special character and every emoji.

**Comment by pipobscure** [comment]
This should be fixed with fsevents@2.0.7 please try reinstalling to verify!

**Comment by fabiospampinato** [comment]
I can confirm this is fixed, no more errors with special characters (I've tried `∕`, `İ` and `👍`)
